packages/cozy-harvest-lib/src/components/cards/LaunchTriggerAlert.jsx

```diff
@@ -27,9 +27,11 @@ const useStyles = makeStyles({
   root: {
     padding: '.5rem 1rem'
   },
-  message: {
-    maxWidth: ({ block }) => block && 'calc(100% - 16px - .5rem)' // 16px is the size of the icon
-  },
+  message: ({ block }) =>
+    block && {
+      maxWidth: 'calc(100% - 16px - .5rem)', // 16px is the size of the icon
+      width: '100%'
```

Comment:
I don't think this change is needed as long as Cozy UI is up to date?
This was the point of the `flex: auto;` added in cozy/cozy-ui#2333.